code/modules/antagonists/cult/cult.dm

```diff
@@ -295,7 +295,7 @@
 /proc/is_convertable_to_cult(mob/living/target, datum/team/cult/specific_cult)
 	if(!istype(target))
 		return FALSE
-	if(isnull(target.mind) || !GET_CLIENT(target))
+	if(isnull(target.mind))// || !GET_CLIENT(target))
```

Comment:
This does have a reason to exist, the reason why it's `GET_CLIENT` is so you CAN test stuff with it by making a mock client

Reply:
it's literally the only traitor panel conversion that needs a client. you can just give it a mock client if you need to beforehand

Reply:
at a glance it looks like the convert rune can certainly call this on mobs with minds but without clients, and i would rather be safe than sorry.